docs/bis/guide/data-organization/information-hierarchy.md

```diff
@@ -0,0 +1,125 @@
+# Information Hierarchy
+
+<!-- TODO: Some of the information in this chapter will likely be moved to [Model Fundamentals](./model-fundamentals.md). It will be natural to have links from this chapter to that one. -->
+
+The information in a BIS repository is arranged in a hierarchy that is governed by rules. Some of the rules are explicitly defined by the schemas, and other rules require the applications that are creating data to follow standards.
+
+The hierarchies in BIS repositories are intended to facilitate both human and software comprehension of the data.
+
+## Hierarchy Constructs
+
+There are several mechanisms available in BIS that domain authors can take advantage of in order to organize elements into hierarchies:
+
+* The Subject hierarchy organizes Models.
+* A Model can contain Elements.
+* An Element can be sub-modeled by a Model.
+* An Element can own child Elements.
+* A *Spatial Organizer* can hold an Element as part of the spatial structure of a *Facility*, from the perspective of a specific discipline.
+* A *Physical System* can group member Elements.
+* The provenance of an Element can be traced back to an *External Source*, whose organization may be hierarchical.
+
+Each of these mechanism is intended to be used in specific circumstances which are explained in this chapter.
+
+### The Subject Hierarchy Organizes Models
+
+The RepositoryModel contains a hierarchy of `Subject` Elements organized into a parent-child hierarchy. All Models (other than the RepositoryModel itself) are organized under these `Subject`s (via InformationPartition)
+
+### Model Contains Elements
+
+A Model is a *container* for Elements.
```

Comment:
Sentences on separate lines with no blank lines in between render as a single paragraph. Was that the intent or were you intending to make a list like under "Hierarchy Constructs"?

Reply:
These particular sentences were not changed wrt their original writing currently in production. No intention to make a list so it is fine that they render as a single paragraph.